easybuild/easyconfigs/n/NCCL/NCCL-2.8.4-1-gcccuda-2020b.eb

```diff
@@ -0,0 +1,24 @@
+# This easyconfig was created by Simon Branford of the BEAR Software team at the University of Birmingham.
+name = 'NCCL'
+version = '2.8.4-1'
+
+homepage = 'https://developer.nvidia.com/nccl'
+description = """The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective
+communication primitives that are performance optimized for NVIDIA GPUs."""
+
+toolchain = {'name': 'gcccuda', 'version': '2020b'}
```

Comment:
I'm thinking about the toolchain for this. Prior we had SYSTEM-CUDA, now (full)gcccuda, which means that it can't be used with Intel toolchains, can it?
So maybe GCCcore with CUDA dep and suffix?